projects/plugins/jetpack/extensions/plugins/publicize/post-publish.js

```diff
@@ -0,0 +1,15 @@
+import {
+	PostPublishManualSharing,
+	PostPublishReviewPrompt,
+} from '@automattic/jetpack-publicize-components';
+
+const PostPublishPanels = () => {
```

Comment:
Extracted the post-publish panels in their own file. Same with the pre-publish panels below.